mrp_multi_level/security/mrp_multi_level_security.xml

```diff
@@ -7,6 +7,16 @@
         <field name="category_id" ref="base.module_category_hidden"/>
     </record>
 
+    <record id="group_mrp_multi_level_run" model="res.groups">
+        <field name="name">Run MRP Manually</field>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]" />
+        <field name="category_id" ref="base.module_category_hidden" />
+    </record>
+
+    <record id="mrp.group_mrp_manager" model="res.groups">
+        <field name="implied_ids" eval="[(4, ref('mrp_multi_level.group_mrp_multi_level_run'))]" />
```

Comment:
This will make any production manager able to run the wizard, ant you cannot unselect it because it is implied... I think it is better if you remove this.

Reply:
Yes, agree, my intention was not to  modify current behavior but that's not ok.